org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/IADTabpanel.java

```diff
@@ -26,6 +30,14 @@
 	public static final String ON_ACTIVATE_EVENT = "onActivate";
 	public static final String ATTR_ON_ACTIVATE_POSTED = "org.adempiere.webui.adwindow.IADTabpanel.onActivatePosted";
 
+	/**
+	 * @param winPanel
+	 * @param windowNo
+	 * @param gridTab
+	 * @param gridWindow
+	 */
+	public void init(AbstractADWindowContent winPanel, int windowNo, GridTab gridTab, GridWindow gridWindow);

```

Comment:
why the need of gridWindow parameter here when you have gridTab.getGridWindow() ?

Reply:
dropped WindowNo and gridWindow from argument.